endpoints/openrtb2/auction.go

```diff
@@ -223,10 +223,14 @@ func (deps *endpointDeps) validateRequest(req *openrtb.BidRequest) error {
 		return err
 	} else if bidExt != nil {
 		aliases = bidExt.Prebid.Aliases
-	}
 
-	if err := deps.validateAliases(aliases); err != nil {
-		return err
+		if err := deps.validateAliases(aliases); err != nil {
```

Comment:
Always good to skip unneeded work (validating aliases or bid adjustments when none are supplied) ASAP :)